tests/unittests/quantizationTest.cpp

```diff
@@ -138,6 +138,35 @@ TEST(Quantization, quantizeGraph) {
   EE.run({}, {});
 }
 
+TEST(Quantization, quantizeReLU) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  auto *input = mod.createVariable(ElemKind::FloatTy, {1, 3}, "input");
+  auto *relu = F->createRELU("ReLU", input);
+  F->createSave("ret", relu);
+
+  // Make sure that offset quantization parameter of ReLU is set
+  // such that it produces non-negative floating point range.
+  std::vector<NodeQuantizationInfo> QI{
+      {NodeQuantizationInfo::generateNodeOutputName(input->getName()),
+       {0.2f, 0}},
+      {NodeQuantizationInfo::generateNodeOutputName(relu->getName()),
+       {0.2f, -128}}};
+
+  F = quantization::quantizeFunction(EE, QI, F);
+
+  EE.compile(CompilationMode::Infer, F);
+
+  ASSERT_TRUE(llvm::isa<MaxNode>(F->getNodeByName("relu1")));
```

Comment:
Where does `"relu1"` come from? Is it hardcoded somewhere not in this test? In general I think it's better practice to access nodes in the compiled graph by going through the Variables, in this case either `"input"` or the SaveNode's Variable. I think that would be less fragile.